Source/ProjectFirma.Web/Views/DNRUplandRegion/DetailViewData.cs

```diff
@@ -43,9 +44,10 @@ public class DetailViewData : FirmaViewData
         public readonly MapInitJson MapInitJson;
         public readonly ViewGoogleChartViewData ViewGoogleChartViewData;
         public readonly List<PerformanceMeasureChartViewData> PerformanceMeasureChartViewDatas;
+        public readonly string Coordinator;

```

Comment:
Could probably remove this var once the field is created as an ID and get the updated generator code